docs/source/conf.py

```diff
@@ -340,3 +341,41 @@
 
 spelling_lang = 'en_US'
 spelling_word_list_filename = 'spelling_wordlist.txt'
+
+
+def _import_object_from_name(module_name, fullname):
+    obj = sys.modules.get(module_name)
+    if obj is None:
+        return None
+    for comp in fullname.split('.'):
+        obj = getattr(obj, comp)
+    return obj
+
+
+def linkcode_resolve(domain, info):
+    if domain != 'py' or not info['module']:
+        return None
+
+    tag = 'v{}'.format(__version__)
```

Comment:
I understand the issue.
In readthedocs, 'latest' points to 'master' branch and continuously updated along with the branch.
If the doc version the user is browsing is 'latest', we need to use 'READTHEDOCS_VERSION' in order to avoid disparity between the link (which would point to the specific tagged commit, if only using `__version__`) and the master content.